lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -21,10 +21,13 @@ Gem::Specification.new do |spec|
   if spec.respond_to?(:metadata)
     spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
   else
-    raise "RubyGems 2.0 or newer is required to protect against public gem pushes."
+    raise "RubyGems 2.0 or newer is required to protect against" \
```

Comment:
please move the space up to this line and squash the change down to a single commit? Thanks!